### DISCUSSION TITLE
Wix upgrade of a BootStrapper built using WixToolset v4 is causing one of the nested and embedded msi built using v3, to get uninstalled

### DISCUSSION THREAD

**Discussion by alawans** [wixtoolset/issues#7974] [Questions]
I am having a Bootstrapper1 built using WixToolset v4 version. This Bootstrapper1 contains multiple msi inside and one another Bootstrapper2 which has been built using WixToolset v3. The Bootstrapper2 contains multiple msi built using v3 only. 
We are having a Major upgrade scenario where Bootstrapper1 level is getting updated to a newer level. The newer level also contains a newer level of Bootstrapper2. This Bootstrapper2 has few msi which are coming to a newer level and few which are remaining the same as previous level. 
I am seeing an issue once the upgrade has completed, the unmodified msi contained in Bootstrapper2 are getting uninstalled and leading to many issues in the system. 
As per the logs it indicates to me that during the upgrade, the unmodifed msi were simply ignored and the rest of the modified msi in both the bootstrappers were performed correctly, but towards the end it is trying to check for the previous versions and do the cleanup of it. During this process it is marking the unmodified msi for uninstall.
I do not for sure know if this is a compatibility issue or not and would like to get some inputs form all. it is to be noted that individually both the bootstrappers are getting upgraded if I initiated manually and separately. Also, this was all working fine when both the bootstrappers were on v3. We moved Boostrapper1 to v4 about 3 months back and since then it was working fine. But we are facing this issue now and have no clue on what is causing the uninstall.
Highly appreciate your inputs and time given on this topic. Thanks

**Comment by robmen** [comment]
An issue was opened tracking this breaking change. We triaged it in one of the recent WiX Online Meetings.

**Reply by alawans** [reply]
Thanks a lot for a quick response to my query. How do access the newly opened issue? Can you please let me know?

**Reply by wmanning** [reply]
Issue repository: https://github.com/wixtoolset/issues
Somewhere in there.
Blog covers all meetings: https://www.firegiant.com/blog/

**Reply by alawans** [reply]
Thanks for sharing the link. I wanted to know the corresponding issue no or meeting details where my issue got discussed. it will help me understand more on what the next steps for me could be to have it resolved.
